README.md

```diff
@@ -1,22 +1,92 @@
-# GetIt
+<div align="center">
 
-## Build Setup
+# 🛒 GETIT
+</div>
+Streamlined college e-commerce: Simplifying merch buying and selling for students.
+
+
+## ✨FEATURES
+
+- Clubs can sell
+- Buy the merch you like
+- Automatic payments for sellers no need for manual verification
+- Seller reports
+
+
+
+
+---
+## 📱 Tech Stack 
+
+[![My Skills](https://skills.thijs.gg/icons?i=vue,nuxt,nodejs)](https://skills.thijs.gg)
+### FRONTEND:
+- [VUE](https://vuejs.org/guide/introduction.html)
+- [NUXT](https://nuxt.com/docs)
+
+### BACKEND: 
+
+TO BE UPDATED
+
+
+## 🔰 GETTING STARTED
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### [![My Skills](https://skills.thijs.gg/icons?i=nodejs)](https://skills.thijs.gg)PREREQUISITS
+
+The following software is required to be installed on your system:
+
+* Node 16.x
+* Npm 8.x
+
+Type the following commands in the terminal to verify your node and npm versions
 
 ```bash
-# install dependencies
-$ npm install
+node -v
+npm -v
+```
 
-# serve with hot reload at localhost:3000
-$ npm run dev
+### ![NPM](https://img.shields.io/badge/NPM-%23CB3837.svg?style=for-the-badge&logo=npm&logoColor=white) BASIC SETUP
 
-# build for production and launch server
+
+#### Follow the steps mentioned below to setup the project locally on your computer
+
+1. Fork the repository by clicking on `Fork` option on top right of the main repository.
+2. Open Command Prompt/Terminal on your local computer.
+3. Clone the forked repository by adding your own GitHub username in place of `<username>`.
+
+```bash
+    git clone https://github.com/<username>/resource-sharing/
+```
+4. Navigate to the resource-sharing directory.
+
+```bash
+    cd resource-sharing
+```
+
+5. Install all resource-sharing dependencies. 
+
+```bash
+    npm install
+```
+
+6. Run the website locally.
+
+```bash
+    npm run dev
+```
+
+7. Access the live development server at [localhost:5173](http://localhost:5173).
+
+8. Build Setup
+```
```

Comment:
also no need for point no 5, 6 and 7 they are mentioned below

Reply:
should  ,i just delete the following points, since they were not mentioned after point 8?

Reply:
yeah you can delete those, no. 8 is enough